packages/azure-services/src/register-azure-services-to-container.ts

```diff
@@ -51,6 +51,14 @@ export function registerAzureServicesToContainer(container: Container): void {
         return createCosmosContainerClient(context.container, 'scanner', 'a11yIssues');
     });
 
+    container.bind(cosmosContainerClientTypes.ScanBatchesCosmosContainerClient).toDynamicValue(context => {
+        return createCosmosContainerClient(context.container, 'scanner', 'scanBatches');
+    });
+
+    container.bind(cosmosContainerClientTypes.ScanRunsCosmosContainerClient).toDynamicValue(context => {
+        return createCosmosContainerClient(context.container, 'scanner', 'scanRuns');
+    });
```

Comment:
this is same as 
 container.bind(cosmosContainerClientTypes.OnDemandPageScanRunResults).toDynamicValue(context => {
        return createCosmosContainerClient(context.container, 'scanner', 'onDemandPageScanRunResults');
    });

Reply:
updated